cmd/sidecars/kafka.go

```diff
@@ -50,19 +48,32 @@ func KafkaEnvConfig(config KafkaConfig) k8s.ContainerOption {
 }
 
 // KafkaSidecarConfig adds the necessary env variables to configure the sidecar in the cluster
-func KafkaSidecarConfig(config KafkaConfig, conns *models.SidecarConnections) k8s.ContainerOption {
+func KafkaSidecarConfig(config KafkaConfig, conns *models.SidecarConnections) (k8s.ContainerOption, []corev1.EnvVar) {
+	port := corev1.EnvVar{
+		Name:  "INSPR_SIDECAR_KAFKA_WRITE_PORT",
+		Value: strconv.Itoa(int(conns.InPort)),
+	}
+
+	kafkaAddr := corev1.EnvVar{
+		Name:  "INSPR_SIDECAR_KAFKA_ADDR",
+		Value: config.KafkaInsprAddr,
+	}
+
 	return k8s.ContainerWithEnv(
-		corev1.EnvVar{
-			Name:  "INSPR_SIDECAR_KAFKA_READ_PORT",
-			Value: string(conns.OutPort),
-		},
-		corev1.EnvVar{
-			Name:  "INSPR_SIDECAR_KAFKA_WRITE_PORT",
-			Value: string(conns.InPort),
-		},
-		corev1.EnvVar{
-			Name:  "INSPR_SIDECAR_KAFKA_PORT",
-			Value: config.KafkaInsprPort,
-		},
-	)
+			port,
+		),
+		[]corev1.EnvVar{port, kafkaAddr}
+}
+
+func returnKafkaContainerOptions(app *meta.App, config KafkaConfig, envVars k8s.ContainerOption,
```

Comment:
i don't understand the necessity of the word `return` in the method name, there is no need to fix this but `kafkaContainerOptions` works as just as fine

Reply:
Maybe something like `getKafkaContainerOptions` its better